packages/web-app-preview/package.json

```
@@ -4,10 +4,15 @@
   "private": true,
   "description": "ownCloud Web Preview",
   "license": "AGPL-3.0",
+  "devDependencies": {
+    "@panzoom/panzoom": "^4.5.1",
+    "web-test-helpers": "workspace:*"
+  },
   "peerDependencies": {
-    "vue3-gettext": "^2.3.3",
-    "vuex": "4.1.0",
     "web-client": "workspace:@ownclouders/web-client@*",
-    "web-pkg": "workspace:@ownclouders/web-pkg@*"
+    "web-pkg": "workspace:@ownclouders/web-pkg@*",
+    "@vueuse/core": "^10.3.0",
+    "vue3-gettext": "2.4.0",
```

I feel like we shouldn't bump any dependencies if not necessary in a patch release.

```suggestion
    "vue3-gettext": "^2.3.3",
```